<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>设置男孩年龄</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxBoy">
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnBoyInc">
         <property name="text">
          <string>Boy长大1岁</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnClassInfo">
         <property name="text">
          <string>类的元对象信息</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>设置女孩年龄</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxGirl">
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnGirInc">
         <property name="text">
          <string>girls长大1岁</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>清空对话框</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="textEdit">
       <property name="documentTitle">
        <string> </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
